dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/ESMappingUtilHelper.java

```diff
@@ -0,0 +1,268 @@
+package com.dotcms.content.elasticsearch.util;
```

Comment:
there is a class for ES util methods 

https://github.com/dotCMS/core/blob/4b32284db0ea826ef6cff4e6afe857593635f1a8/dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ElasticsearchUtil.java

I don't know if we need both class

Reply:
I prefer to keep them separate, because this new helper is focused on fields mappings only